fetch_data.py

```diff
@@ -25,27 +30,41 @@
 import random
 from docopt import docopt
 from pathlib import Path
+from enum import Enum
 
 
 TWITTER_RATE_LIMIT_ERROR = 88
 
 
-def fetch_users(apis, target, are_users, nodes_to_consider, max_tweets_count, out_path,
+class Mode(Enum):
+    USERS = 1
+    TWEETS = 2
```

Comment:
I think `TWEETS` may be ambiguous, since likes are also tweets. Maybe rename it to `SEARCH`?

Reply:
Done.

Note that this Enum was reflecting the mode names from the docopt (`users`, `tweets`, and `likes`). For consistency, my commit making this change also renamed the corresponding entry in the docopt, as well as updating some docstrings which referred to the "tweets" mode. This means that the input mode names are `users`, `search`, and `likes`, which is a behavior change for the CLI. PTAL and let me know if I should revert the changes that impact the CLI.